prow/plugins/approve/approvers/README.md

```diff
@@ -185,27 +194,33 @@ K8s-bot updates comment:
 	You can indicate your approval by writing `/approve` in a comment
 	You can cancel your approval by writing `/approve cancel` in a comment
 
-K8s-bot merges the PR. It needs to wait its turn in submit queue and passes tests.
+K8s-bot merges the PR, because it has both the **lgtm** and **approved**. It K8s-bot still needs to wait its turn in submit queue and pass tests.
```

Comment:
It is probably worth noting that this assumes Tide is deployed and configured to require `lgtm` and `approved` labels for merges to that repo. Tide configuration guide: https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/config.md

Reply:
Updated and linked.